handsontable/src/editors/textEditor/textEditor.js

```diff
@@ -475,8 +475,6 @@ export class TextEditor extends BaseEditor {
       runOnlyIf: event => !this.hot.selection.isMultiple() && // We trigger a data population for multiple selection.
         // catch CTRL but not right ALT (which in some systems triggers ALT+CTRL)
         !event.altKey,
-      relativeToGroup: EDITOR_MANAGER_GROUP,
-      position: 'before',
```

Comment:
@wszymanski I have a question about these lines. Are they still necessary? While I reorganized the keyboard shortcuts and moved them to a separate module, I saw that after removing these lines cell editor still worked as usual. Moreover, there are no failing tests.

Reply:
I can't see any difference after selecting single cells, opening an editor and using `control+enter`. Maybe it has been copied unintentionally from another part of the code? Maybe @AMBudnik will know some specific case?

Reply:
The [code sits](https://github.com/handsontable/handsontable/blob/develop/handsontable/src/editors/textEditor/textEditor.js#L478-L479) on the `develop` branch since the ShortcutManager introduction. If editors work without that code, I'd leave them removed.

Reply:
I can see that tests are passing for 3 places in the code when similar action is used (`control+enter`, `meta+enter`, `alt+enter`). I can't find any case which shows a difference (after removing those lines).